hi-ml-cpath/src/health_cpath/scripts/generate_crossval_report.py

```diff
@@ -242,6 +252,9 @@ def render_confusion_matrices(report: HTMLReport, heading: str, level: int, clas
                                                                     "in the generated report.")
     parser.add_argument('--overwrite', action='store_true', help="Forces (re)download of metrics and output files, "
                                                                  "even if they already exist locally.")
+    parser.add_argument("--hyper_arg_name", default="crossval_index",
+                        help="Name of the Hyperdrive argument used for indexing the child runs.")
+    parser.add_argument("--primary_metric", default="auroc", help="Name of the reference metric to optimise.")
```

Comment:
can we use `MetricsKey.AUROC` as default?

Reply:
Done!